include.js

```diff
@@ -13,6 +13,13 @@ let cards = [
     author: 'Ibaad',
     githubLink: 'https://github.com/ibaaddurrani'
   }
```

Comment:
```suggestion
  },
```
There needs to be a comma, otherwise it's not a valid Javascript array.